sonar-db/src/main/java/org/sonar/db/measure/MeasureDto.java

```diff
@@ -190,13 +189,11 @@ public MeasureDto setSnapshotId(Long snapshotId) {
     return this;
   }
 
-  @CheckForNull
-  public Integer getRuleId() {
-    return ruleId;
-  }
-
+  /**
+   * @deprecated in 5.5. Does nothing. Kept for compatibility with developer cockpit plugin, version 1.10
+   */
+  @Deprecated
   public MeasureDto setRuleId(@Nullable Integer ruleId) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)